src/custom-element.js

```diff
@@ -1716,6 +1730,27 @@ function createBaseCustomElementClass(win) {
       });
     }
 
+    insertSandboxAnalytics_() {
+      this.sandboxAnalyticsConfigPromise_ =
+          this.implementation_.createSandboxAnalyticsConfigCallback();
+      if (this.sandboxAnalyticsConfigPromise_) {
+        this.sandboxAnalyticsConfigPromise_.then(config => {
+          this.sandboxAnalyticsConfigPromise_ = null;
+          this.sandboxAnalyticsElement_ = insertAnalyticsElement(
```

Comment:
I see your point. But `insertAnalyticsElement` was in core binary before, in `src/analytics.js`.
FYI I get the size:
master: 
  1.14 MB   |   217.61 kB   |                             65.44 kB   |   v0.js / amp.js
After this PR:
  1.13 MB   |   218.48 kB   |                             65.64 kB   |   v0.js / amp.js